libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/app/SalesforceReactSDKManager.java

```diff
@@ -46,44 +48,44 @@
 
 import java.util.ArrayList;
 import java.util.Collections;
-import java.util.LinkedHashMap;
 import java.util.List;
+import java.util.Map;
 
 /**
  * SDK Manager for all react native applications
  */
 public class SalesforceReactSDKManager extends MobileSyncSDKManager {
 
-	/**
-	 * Protected constructor.
-	 *
-	 * @param context Application context.
-	 * @param mainActivity Activity that should be launched after the login flow.
-	 * @param loginActivity Login activity.
-	 */
-	protected SalesforceReactSDKManager(Context context, Class<? extends Activity> mainActivity,
+    /**
```

Comment:
I avoided whole-file formatted where possible, but this one was really inconsistent and it was challenging to get a readable diff without correcting the indentation.